lib/rules/no-jquery-selector.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @fileOverview Disallow the use of jQuery selector.
+ */
+'use strict';
+
+const { get } = require('../utils/get');
+const componentElementMessage = 'The use of this.$() to get a component\'s element is deprecated. Use \'this.element\' instead';
```

Comment:
This should like be something like `Use 'this.element.querySelector' or a similar API instead.`

Reply:
So I think that might be too specific.

Devs don't always do `this.$(selector)`, but sometimes `this.$().focus()`, `this.$().on(...)`, `this.$().remove()` etc

Reply:
Ah, good point. Seems good then.